api/notd/store/saver.py

```diff
@@ -60,7 +58,7 @@ async def create_token_transfer(self, retrievedTokenTransfer: RetrievedTokenTran
             blockDate=retrievedTokenTransfer.blockDate,
         )
 
-    async def create_token_metadata(self, tokenId: int, registryAddress: str, metadataUrl: str, imageUrl: Optional[str], name: Optional[str], description: Optional[str], attributes: Optional[JSON]) -> TokenMetadata:
+    async def create_token_metadata(self, registryAddress: str, tokenId: str, metadataUrl: Optional[str] = None, description: Optional[str] = _EMPTY_STRING, imageUrl: Optional[str] = _EMPTY_STRING, name: Optional[str] = _EMPTY_STRING, attributes: Optional[str] = _EMPTY_OBJECT) -> TokenMetadata:
```

Comment:
this is wrong i think you should change it back. the empty string and objects are only there to protect when setting null